### PR TITLE
Added Features

### PR DESCRIPTION
**Pull request by Shanzid01**
Feature update
-added Google Analytics to landing page
-added Get In Touch
    ---user input is forwarded to **thersquaredproject@gmail.com** (used formspree.io)
    ---form validates before submit (i.e. no empty fields allowed)
    ---Google reCaptcha enabled to prevent spam submissions
    ---UI is fully responsive

Fixed bugs
-fixed some css typing errors in index.css

### CODE REVIEW

File: front-end-master/Projects/projects.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+html{
```
Comment (azmainamin) [inline front-end-master/Projects/projects.css:1]: Great job using relative values.

File: front-end-master/Projects/projects.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+.project .title > h1{
+	font-family: 'Raleway', sans-serif;
+	position: relative;
+	font-weight: 600;
+	padding: 2vw;
+	font-size: 2.8em;
+}
+.project .title h1::after{
+	content: '';
```
Comment (azmainamin) [inline front-end-master/Projects/projects.css:38]: Do we need this?

File: front-end-master/about_us/about.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-class Division{
-	 constructor{
-
-	 }
-}
+//class Division{
```
Comment (azmainamin) [inline front-end-master/about_us/about.js:1]: If we dont need this, please remove the commented out lines.

File: front-end-master/landing_page/icons/desktop.ini [hunk 4]
```diff
@@ -1,4 +0,0 @@
-[.ShellClassInfo]
-InfoTip=This folder is shared online.
```
Comment (azmainamin) [inline front-end-master/landing_page/icons/desktop.ini:2]: We don't want to commit desktop.ini files

File: front-end-master/landing_page/index.css [hunk 5]
```diff
@@ -404,31 +396,108 @@ a, a:hover{
...
+.card{
+	box-shadow: 0 2px 2px 0 rgba(0,0,0,0.14), 0 3px 1px -2px rgba(0,0,0,0.12), 0 1px 5px 0 rgba(0,0,0,0.2);
+}
+.contact-heading{
+	margin-bottom:30px; 
+}
+.contact-heading span:nth-of-type(1){
+	display: block;
+	font-family: 'Raleway';
+	font-size: 40px;
```
Comment (azmainamin) [inline front-end-master/landing_page/index.css]: Can we use relative units instead of absolute ones?

File: front-end-master/landing_page/index.html [hunk 6]
```diff
@@ -171,33 +170,55 @@ <h1 class="headline">Our Associates</h1>
...
+                        <div class="col-12">
+                            <textarea class="form-control" rows="3" name="Message" required="required" placeholder="Write a message.."></textarea>
+                        </div>
                     </div>
-                    <button class="btn">Submit</button>
-                </form>
-            </div>
-
-            <div class="form-sidebar">
+                    <div class="form-row" style="margin-top: 20px">
```
Comment (azmainamin) [inline front-end-master/landing_page/index.html]: Any reason why this style is inline? We should be consistent and move it out to a css file.

File: front-end-master/Projects/projects.html [hunk 7]
```diff
@@ -0,0 +1,102 @@
+<!doctype html>
```
Comment (azmainamin) [inline front-end-master/Projects/projects.html:1]: Why are all these lines green? It doesn't seem that these html files are relevant to the work. I suspect something happened with Git.

File: front-end-master/meet_the_team/team.css [hunk 8]
```diff
@@ -1,38 +0,0 @@
-html,body
```
Comment (azmainamin) [inline front-end-master/meet_the_team/team.css:1]: And again, why are these lines removed? This has nothing to do with the contact us forms.

**Review by azmainamin — CHANGES_REQUESTED** [review]

**Comment by azmainamin** [comment]
Also, I couldn't test this out since we are not in a server yet, but when the user successfully sends an email, will get some sort of message saying Succes? And when it fails, will the user get an error message?

**Comment by Shanzid01** [comment]
Most of the issues you pointed out wasn't the code written by me.

File: front-end-master/landing_page/index.html (hunk 6 above)
Comment (Shanzid01) [reply front-end-master/landing_page/index.html]: since it was a one-time thing I needed to do so I just decided to write it inline instead of overwriting the "form-row" class and having the changes everywhere.

File: front-end-master/landing_page/index.css (hunk 5 above)
Comment (Shanzid01) [reply front-end-master/landing_page/index.css]: sure!

**Comment by Shanzid01** [comment]
I made the changes you requested for. But only in the places where I've written the code. I think the rest should be looked at by Nabil since he wrote the code and he'd understand that section better.

File: front-end-master/landing_page/index.css [hunk 9]
```diff
@@ -409,7 +409,7 @@ a, a:hover{
 
 /*------------------------------*/
 .contact-card-container{
-	max-width: 710px !important;
+	max-width: 700px !important;
```
Comment (azmainamin) [inline front-end-master/landing_page/index.css]: I don't like the use of !important. Is this absolutely necessary?

**Review by azmainamin — APPROVED** [review]

File: front-end-master/landing_page/index.css (hunk 9 above)
Comment (Shanzid01) [reply front-end-master/landing_page/index.css]: yes, I'm overriding the original class.
